site/profile/manifests/app/pipelines/agent.pp

```diff
@@ -1,15 +1,28 @@
 class profile::app::pipelines::agent (
   String $access_token = 'fake',
   String $secret_key = 'fake',
+  String $download_url = 'fake',
```

Comment:
Code cleanliness: for required parameters, we should not provide a default value. That will eliminate the need to specify "fake" here, or "ENTER_VALUE" in common.yaml.